publish.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+#
+# This script can be executed to publish all results found in `result` subdir. It will
+# compress the loader image because Capstan requires compressed file when pulling 
+# images from the repository. Then it will look for all package files and their manifests
+# (*.mpm and *.yaml) and upload them sequentially.
+#
+# Before using this script, you have to configure s3cmd (http://s3tools.org/s3cmd).
+# After installing, you can invoke the configuration command
+#
+# $ s3cmd --configure
+#
+# You will be asked few AWS related questions (region, key, secret) that you need to
+# provide for the user with write access.
+
+pushd result
+
+echo "Compressing OSv loader image"
+gzip osv-loader.qemu
+echo "Uploading OSv loader image"
+s3cmd put --acl-public --guess-mime-type osv-loader.qemu.gz s3://mikelangelo-capstan/mike/osv-loader/osv-loader.qemu.gz
```

Comment:
Oh, I didn't answer Q about providing your own version of index.yaml: I think we better automatically provide it. At the moment it's hard-coded into main python program, but it shouldn't be too difficult to dynamically query date and osv version.

Bottomline, I'd suggest we don't do it in publish.sh script (that will be run from host) but rather in capstan-packages.py (that is run inside conatiner and thus can access actual OSv version and everything).